tests/put/test_end_to_end_put.py

```diff
@@ -25,43 +25,46 @@ def test_without_args(self):
 
         assert [first_line_of(result.stderr),
                 result.exit_code] == \
-               ['Usage: trash-put [OPTION]... FILE...', 2]
+               ['usage: trash-put [OPTION]... FILE...', 2]
 
     def test_wrong_option(self):
-        result = run_command.run_command(self.tmp_dir, "trash-put", ['--wrong-option'])
+        result = run_command.run_command(self.tmp_dir, "trash-put", ['--wrong-option', 'file'])
 
         assert [last_line_of(result.stderr),
                 result.exit_code] == \
-               ['trash-put: error: no such option: --wrong-option', 2]
+               ['trash-put: error: unrecognized arguments: --wrong-option', 2]
 
     def test_on_help(self):
```

Comment:
There is a problem with this test, it fails with python 3.7 as you can see [here](https://github.com/andreafrancia/trash-cli/actions/runs/3076360743/jobs/4970739972)

To fix this please use the `normalize_options()` function like it is done in [`tests/list/test_end_to_end_list.py`](https://github.com/andreafrancia/trash-cli/blob/408ed0cbe40cd83df3c1b094f555cf685f3e959a/tests/list/test_end_to_end_list.py#L81)

Reply:
Fixed.